tests/slow/src/test/scala/tests/mill/MillServerCodeLensSuite.scala

```diff
@@ -14,8 +18,15 @@ class MillServerCodeLensSuite
 
   override def munitTimeout: Duration = Duration("4min")
 
+  override def afterEach(context: AfterEach): Unit = {
+    super.afterEach(context)
+    scribe.info(workspace.resolve(".metals/bsp.trace.json").text)
```

Comment:
Can you print it only if the test fails?

Reply:
I think it's better not to print at all as writing to bsp.trace.json might slow down tests. I will remove it afterwards.

Reply:
Ach and it's typical, when adding logs it works :sweat_smile: